helm/kubernetes-power-manager-ocp-4.13-v2.3.1/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: kubernetes-power-manager-ocp-4.13-v2.3.1
+description: A Helm chart for Intel's Kubernetes Power Manager v2.3.1 using OCP v4.13
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 1.0.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "v2.3.1"
```

Comment:
Tiny nitpick here - update this to "v2.3.1-ocp-4.13" (or "v2.3.1-ocp_4.13", whichever you've been using) so it doesn't clash with the default version's chart name

Reply:
Updated